__tests__/app.test.js

```diff
@@ -149,6 +149,37 @@ describe('/api/reviews/:review_id/comments', () => {
     });
 })
 
+describe('/api/reviews/:review_id/comments', () => {
+    test('POST- status:201- responds with the added comment object', () => {
+        const newComment = { username: 'bainesface', body: 'this is the story about a pickle' }
+        return request(app).post('/api/reviews/1/comments').send(newComment).expect(201).then(({ body }) => {
+            expect(body.newComment).toBeInstanceOf(Object)
+            expect(body.newComment.author).toBe('bainesface')
+            expect(body.newComment.body).toBe('this is the story about a pickle')
+        })
+    })
+
+    test('POST- status:404- when passing a author that doesnt exist in db', () => {
+        const newComment = { username: 'pickle', body: 'this is the story about a pickle' }
+        return request(app).post('/api/reviews/1/comments').send(newComment).expect(404).then(({ body }) => {
+            expect(body.message).toBe('not found')
+        })
+    })
+
+    test('POST- status:404- when passing a review_id that doesnt exist in db', () => {
+        const newComment = { username: 'pickle', body: 'this is the story about a pickle' }
+        return request(app).post('/api/reviews/1000/comments').send(newComment).expect(404).then(({ body }) => {
+            expect(body.message).toBe('not found')
+        })
+    })
+
+    test('POST- status:400- when passing a incorrect body', () => {
+        const newComment = { username: 'bainesface' }
+        return request(app).post('/api/reviews/1/comments').send(newComment).expect(400).then(({ body }) => {
+            expect(body.message).toBe('missing required information')
```

Comment:
this can just be bad request if you want it to be more generic so it can be used elsewhere